ibm_catalog.json

```diff
@@ -173,11 +173,11 @@
             {
               "key": "watsonx_assistant_plan",
               "type": "string",
-              "default_value": "do not install",
+              "default_value": "Do not install",
               "options": [
                 {
-                  "displayname": "do not install",
-                  "value": "do not install"
+                  "displayname": "Do not install",
+                  "value": "Do not install"
```

Comment:
I think value is referenced in the code. It should be `do not install`.  Right @andreainnocenti ?

Reply:
right

Reply:
But is this visible to users? @mkrudele @andreainnocenti 
There's no way to at least style the string correctly with capitalization?